### ISSUE TITLE
It's not possible anymore to write text with label[ ... ] over item images or inventory slots in formspecs

### ISSUE THREAD

**Issue by Miner59** [minetest/minetest#4844]
This would be nice if this would work again.
I found issue  #3672 which fixes the text on the item image button itself. But I think other text still can't overlap images on image buttons or inventory slots. It seems inventory slots now always hide text written over it. I think it should depend on the order the elements are written in the formspec string, which element overlaps the other.

For comparison:

On actual minetest version:
![hiddennumbers](https://cloud.githubusercontent.com/assets/10222419/20868857/63be0f0c-ba66-11e6-9212-3067395764c4.png)

On older version:
http://i58.tinypic.com/w172bt.png

What I talk about are the numbers for counting the items which you can see were working in older versions. I point on the position the number should be with the cursor in upper image.
The code used to do this is the same in both images, it uses label[ ... ] to display numbers over the pictures of the items.

**Comment by Miner59** [comment]
Hello,
I want to bump this issue because I got no response on it.
I made a better screenshot and wrote a little test mod to give better impression of the current behavior.
In older versions, before changes from RBA on generating item images, text written with label in formspecs was in front of images and inventory slots. Now it is always hidden behind.
Truely buggy is the interaction between image buttons and text written with label.
The text is in front of the button, but behind the image depicted on the button. (???)

![example](https://cloud.githubusercontent.com/assets/10222419/21290874/eb9959bc-c4cb-11e6-83be-16b0d31fc26d.png)

test mod:
load the mod, then type /label_test in chat and watch your inventory screen
[label_test.zip](https://github.com/minetest/minetest/files/659194/label_test.zip)
